packages/fxa-graphql-api/src/test/lib/datasources/authServer.spec.ts

```diff
@@ -1,21 +1,22 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Only other vague grumble I might have, is that I wish these `*.spec.ts` modules lived in the same directories as the `*.ts` modules they're testing. But, that's fully preference and a late bit of feedback since we're down the road a bit with this file structure

Reply:
Ah, I'd be good with that, having to navigate to the test mirror directory with all the spec files is annoying, and harder to see if you've missed tests for something than having them right there like you mention.